packages/components/src/site-thumbnail/style.scss

```
@@ -5,6 +5,12 @@
 	justify-content: center;
 	position: relative;
 	overflow: hidden;
+	-webkit-transition: background-color 1s ease-out;
+  	-moz-transition: background-color 1s ease-out;
+  	-o-transition: background-color 1s ease-out;
+  	transition: background-color 1s ease-out;
+	border: 1px solid rgb( 238, 238, 238 );
+  	box-sizing: border-box;
```

Just a little white space thing here

```suggestion
	-webkit-transition: background-color 1s ease-out;
	-moz-transition: background-color 1s ease-out;
	-o-transition: background-color 1s ease-out;
	transition: background-color 1s ease-out;
	border: 1px solid rgb( 238, 238, 238 );
	box-sizing: border-box;
```